mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/master/resourcecluster/TaskExecutorRegistration.java

```diff
@@ -36,36 +37,36 @@
  * Different fields help identify the task executor in different dimensions.
  */
 @Builder
-@FieldDefaults(makeFinal=true, level= AccessLevel.PRIVATE)
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
```

Comment:
Can you remind me why this is not on "@value" tag?

Reply:
because I want to support deserializing two types of TaskExecutorRegistrations. It is not possible to do this with the value tag.